tests/api/GetMetadataApiCest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * ownCloud - mediametadata
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author Jalpreet Singh Nanda (:imjalpreet) <jalpreetnanda@gmail.com>
+ * @copyright Jalpreet Singh Nanda (:imjalpreet) 2016
+ */
+
+use Page\MediaMetadata as MetadataApp;
+
+/**
+ * Class GetMetadataApiCest
+ */
+class GetMetadataApiCest {
+	private $apiUrl;
+	private $params = [
+	];
+	private $metadata = [
+
+	];
+
+	/**
+	 * @param ApiTester $I
+	 */
+	public function _before(ApiTester $I) {
+		$this->apiUrl = MetadataApp::$URL . '/api/metadata';
+	}
+
+	public function _after(ApiTester $I) {
+	}
+
+	public function getMetadataAsUser(\Step\Api\User $I) {
+		$I->am('an App');
+		$I->wantTo('retrieve the metadata stored in the database');
+
+		$data = $I->getFilesDataForFolder('');
+		$params = $this->params;
+
+		$id = $data['testimage.jpg']['id'];
+		$this->getMetadata($I, $params, $id);
+	}
+
+	/**
+	 * @param \Step\Api\User $I
+	 * @param $params
+	 * @param int $id
+	 */
+	private function getMetadata(\Step\Api\User $I, $params, $id) {
+		$I->getUserCredentialsAndUseHttpAuthentication();
+		$params['fileList'] = strval($id);
+		$I->sendGET($this->apiUrl, $params);
```

Comment:
Okay, I will do all the required changes that you have mentioned. But I wanted to ask what is the error that is being shown in Travis, it something different which was not coming earlier.